src/ngx_lua_resty_lmdb_module.c

```diff
@@ -213,9 +213,9 @@ ngx_lua_resty_lmdb_init_conf(ngx_cycle_t *cycle, void *conf)
     }
 
     if (lcf->key_data.data != NULL) {
-        cipher = EVP_get_cipherbyname((char *)lcf->encryption_mode.data);
+        g_cipher = EVP_get_cipherbyname((char *)lcf->encryption_mode.data);
```

Comment:
Actually, I have not seen any Nginx code use the `g_` prefix static variable. It seems like some CPP code style and GNU C code style. But it does not matter.

Reply:
Yes, this var `g_cipher` is inspired by another PR. https://github.com/Kong/lua-kong-nginx-module/pull/60